<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_0">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to our Credit Card Recommender!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Please choose a weightage for each credit card category to help us learn about your preferences. If you have no preferences just click next!</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="gasLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gas</string>
      </property>
     </widget>
     <widget class="QLabel" name="restaurantsLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Restaurants</string>
      </property>
     </widget>
     <widget class="QLabel" name="departmentStoresLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Department Stores</string>
      </property>
     </widget>
     <widget class="QSlider" name="gasSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="restaurantsSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="departmentStoresSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="groceriesSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="groceriesLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Groceries</string>
      </property>
     </widget>
     <widget class="QSlider" name="superMarketsSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="supermarketsLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Supermarkets</string>
      </property>
     </widget>
     <widget class="QLabel" name="wholesaleLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>240</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Wholesale</string>
      </property>
     </widget>
     <widget class="QSlider" name="wholesaleSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gasVal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="restaurantsVal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="departmentStoresVal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="supermarketsVal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>240</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="hotelLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hotel</string>
      </property>
     </widget>
     <widget class="QLabel" name="hotelVal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="hotelSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="airfareLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Airfare</string>
      </property>
     </widget>
     <widget class="QLabel" name="airfareVal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="airfareSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>310</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="groceriesVal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="wholesaleVal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="travelLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Travel</string>
      </property>
     </widget>
     <widget class="QLabel" name="travelVal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>340</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="travelSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="amazonVal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="amazonSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>360</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="amazonLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Amazon</string>
      </property>
     </widget>
     <widget class="QLabel" name="paypalVal">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="paypalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="paypalLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>390</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Paypal</string>
      </property>
     </widget>
     <widget class="QLabel" name="onlineVal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>390</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="onlineSlider">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>410</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="onlineLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>390</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Online</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalWeights">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>440</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="fileUploadButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload .csv</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>39</x>
        <y>130</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Please upload a CSV file containing your credit card or debit card transactions from the past month</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="goToPage2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Congrats here are the top 3 credit cards for you!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Start over!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>510</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressBarStatus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="goToPage1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
